src/ImageSharp/Metadata/ImageMetaData.cs

```diff
@@ -0,0 +1,130 @@
+﻿// <copyright file="ImageMetaData.cs" company="James Jackson-South">
+// Copyright (c) James Jackson-South and contributors.
+// Licensed under the Apache License, Version 2.0.
+// </copyright>
+
+namespace ImageSharp
+{
+    using System.Collections.Generic;
+    using System.Diagnostics;
+
+    /// <summary>
+    /// Encapsulates the metadata of an image.
+    /// </summary>
+    public sealed class ImageMetaData
+    {
+        /// <summary>
+        /// The default horizontal resolution value (dots per inch) in x direction.
+        /// <remarks>The default value is 96 dots per inch.</remarks>
+        /// </summary>
+        public const double DefaultHorizontalResolution = 96;
```

Comment:
They are public so that's why I did not remove them. I did not use auto properties with HorizontalResolution  because I want to protect them from being set to a value lower or equal to zero. I am okay with removing the constants and setting it in the constructor. Can I go ahead and change this?